client/src/components/common/applicant-info-1.vue

```diff
@@ -539,18 +530,15 @@ export default class ApplicantInfo1 extends Vue {
       return event
     }
   }
+  mutateApplicant (key, value) {
+    newReqModule.mutateApplicant({ key, value })
+  }
   mutateCorpNum (num) {
     newReqModule.mutateCorpNum(num)
   }
   queryAddress (id) {
     newReqModule.getAddressDetails(id)
   }
-  showNextTab () {
-    newReqModule.mutateSubmissionTabComponent('ApplicantInfo2')
-  }
-  showPreviousTab () {
-    newReqModule.mutateSubmissionTabComponent('NamesCapture')
-  }
```

Comment:
Unused code

Reply:
If you were keen, you could also delete this from applicant-info-2 and applicant-info-3. :)

Reply:
oh right, will do!